Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -0,0 +1,1375 @@
+            else if (filledQty > 0 && pendingQty > 0)
+            {
+                return OrderStatus.PartiallyFilled;
+            }
+            else if (pendingQty == 0)
+            {
+                return OrderStatus.Filled;
+            }
+            else if (orderDetails.Status.ToUpperInvariant() == "CANCELLED")
+            {
+                return OrderStatus.Canceled;
+            }
+
+            return OrderStatus.None;
+        }
+
+
+
+        /// <summary>
+        /// Gets all open positions
+        /// </summary>
+        /// <returns></returns>
+        public override List<Holding> GetAccountHoldings()
+        {
+            var holdingsList = new List<Holding>();
+            if (_zerodhaProductType.ToUpperInvariant() == KiteProductType.MIS.ToString().ToUpperInvariant())
+            {
+                var HoldingResponse = _kite.GetPositions();
+                if (HoldingResponse.Day.Count == 0)
+                {
+                    return holdingsList;
+                }
+                foreach (var item in HoldingResponse.Day)
+                {
+
+                    Holding holding = new Holding
+                    {
+                        AveragePrice = item.AveragePrice,
+                        Symbol = _symbolMapper.GetLeanSymbol(item.TradingSymbol),
+                        MarketPrice = item.LastPrice,
+                        Quantity = item.Quantity,
+                        Type = SecurityType.Equity,
+                        UnrealizedPnL = item.Unrealised,
+                        CurrencySymbol = Currencies.GetCurrencySymbol(AccountBaseCurrency),
+                        MarketValue = item.ClosePrice * item.Quantity
+
+                    };
+                    holdingsList.Add(holding);
+                }
+
+            }
+            else if (_zerodhaProductType.ToUpperInvariant() == KiteProductType.CNC.ToString().ToUpperInvariant())
+            {
+                var HoldingResponse = _kite.GetHoldings();
+                if (HoldingResponse == null)
+                {
+                    return holdingsList;
+                }
+                foreach (var item in HoldingResponse)
+                {
+
+                    Holding holding = new Holding
+                    {
+                        AveragePrice = item.AveragePrice,
+                        Symbol = _symbolMapper.GetLeanSymbol(item.TradingSymbol),
+                        MarketPrice = item.LastPrice,
+                        Quantity = item.Quantity,
+                        Type = SecurityType.Equity,
+                        UnrealizedPnL = item.PNL,
+                        CurrencySymbol = Currencies.GetCurrencySymbol(AccountBaseCurrency),
+                        MarketValue = item.ClosePrice * item.Quantity
+
+                    };
+                    holdingsList.Add(holding);
+                }
+            }
+            else
+            {
+                //TODO: Pending for the NRML product type
```

Comment:
Not sure if this is planned to be added later in separate PR ?

Reply:
Fixed.